src/content/development/contribution-guide/_index.en.md

```diff
@@ -10,23 +10,38 @@ in order for their contributions to the project to be addressed in a timely mann
 
 ## Project Resources
 
-To be expanded
+Submariner uses [GitHub Projects] to manage releases. Read [Tracking progress on your project board] to learn more on how to work with 
+projects.
 
 ### Backlog Board
 
-To be expanded
+[The backlog board] hosts issues or features that are not scheduled for the release currently being developed. While each issue is opened in
```

Comment:
```suggestion
[The backlog board] hosts issues or features that are not scheduled for the release currently being developed. While each issue is opened in
```

This is not true anymore, right? We're now keeping things on the backlog board, with `next-version-candidate` and prio labels, and letting devs move them to 0.13 I think?

Reply:
Yes this content needs to be updated that any small/medium tasks are kept in this board and moved to the project board when devs pick them up, which is done in the order or priority.